site/content/docs/03-run-time.md

```diff
@@ -558,10 +558,29 @@ A `spring` store gradually changes to its target value based on its `stiffness`
 
 ---
 
-As with [`tweened`](/docs#run-time-svelte-motion-tweened) stores, `set` and `update` return a Promise that resolves if the spring settles. The `store.stiffness` and `store.damping` properties can be changed while the spring is in motion, and will take immediate effect.
+The `store.stiffness`, `store.damping` and `store.precision` properties can be changed while the spring is in motion, and will take immediate effect.
```

Comment:
Since this is directly after the bullet points, we could probably say this instead to not repeat ourselves.

```suggestion
All of the options above can be changed while the spring is in motion, and will take immediate effect.
```

Note: I use 'options' because the previous paragraph also refers them as "The following options are ..."